web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/AspaceRefIdControllerTest.java

```diff
@@ -158,7 +178,7 @@ public void importRefIdsSuccess() throws Exception {
 
```

Comment:
It would be good to update one of the existing import tests to pass in a CSV in the export format to verify it works in a round trip